builds/vs2015/PlayerLib.vcxproj

```diff
@@ -752,4 +756,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
+</Project>
```

Comment:
Is okay then, I just thought it was MSVC that added it in #1100. I am just against unnecessary whitespace changes and this case is likely to get changed over and over again.